src/main/java/frc/robot/commands/ArmDefaultCommand.java

```diff
@@ -1,19 +1,19 @@
 package frc.robot.commands;
 
-import edu.wpi.first.wpilibj.XboxController;
 import edu.wpi.first.wpilibj2.command.CommandBase;
 import frc.robot.Constants.ArmConstants;
-import frc.robot.Constants.OperatorConstants;
 import frc.robot.subsystems.ArmSystem;
-import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
+import frc.robot.util.AppliedController;
 
 public class ArmDefaultCommand extends CommandBase {
     // Goal is to get joystick input and turn that into what we want to set the arm to
-    private XboxController m_Controller;
-    private ArmSystem m_arm = new ArmSystem();
+    private AppliedController m_controller;
```

Comment:
not sure what the difference is between AppliedController and XboxController

Reply:
It is the same as Xbox controller but automatically applied deadband. I will put extra documentation in the code for this.